modules/ROOT/pages/index.adoc

```diff
@@ -30,6 +30,8 @@ Click the Helpful Links item in the left navigation area for links to documentat
 
 Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.
 
+Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.
```

Comment:
@wwarner-salesforce , @mcampo :
this is good, but I think Language support should be a separate section/heading by itself. I don't know where or under which sub-heading. Any thoughts?

Reply:
I think the best option is to add a small section on the front page near the "About Content" section. I'll implement that now in this draft.

A second option is to add it to the "Create an Asset" page: https://docs.mulesoft.com/exchange/to-create-an-asset

Reply:
Here's the separate section on the front page. Please let me know what you think.